ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html

```diff
@@ -21,18 +21,21 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
       <mat-form-field>
         <input matInput type="text" [(ngModel)]="participantId" placeholder="Participant ID" maxlength="200" style="width: 10cm">
       </mat-form-field>
-      <mat-checkbox color="primary" disableRipple
-                   [checked]="deletedFromDDP" (change)="deletedFromDDP = $event.checked">Delete from DDP</mat-checkbox>
+<!--      <mat-checkbox color="primary" disableRipple-->
```

Comment:
Do you typically leave these items in code, commented out?

Reply:
Well, usually no, but it was suggested in the ticket